### PR TITLE
Sidebar improvements

### PR DESCRIPTION
**Pull request by lukeac123**

### CODE REVIEW

File: packages/site-components/src/VerticalNavigation.tsx [hunk 1]
```diff
@@ -17,11 +17,26 @@ export type VerticalNavigationProps = {
 const MenuIcon = ({ open }) => <Icon name={open ? 'chevronDown' : 'chevronRight'} />;
 
 const menuItemStyles = {
-  button: ({ active }) => {
-    return {
-      fontWeight: active ? 'var(--fontWeight-bold)' : 'var(--fontWeight-regular)'
-    };
+  root: {
+    fontSize: '13px'
```
Comment (lukeac123) [inline packages/site-components/src/VerticalNavigation.tsx:21]: needs to be plugged into vanilla extract theme

File: packages/site-components/src/VerticalNavigation.tsx [hunk 2]
```diff
@@ -17,11 +17,26 @@ export type VerticalNavigationProps = {
...
+  },
+  subMenuContent: {
+    backgroundColor: 'inherit'
+  },
+  button: ({ active }) => ({
+    fontWeight: active ? 'var(--fontWeight-bold)' : 'var(--fontWeight-regular)',
+    '&:hover': {
+      opacity: 0.5,
```
Comment (lukeac123) [inline packages/site-components/src/VerticalNavigation.tsx:29]: temporary fix - would useColorMode() should be used to switch on hover colours between light and dark modes but not working in this file for some reason

File: packages/theme/src/sidebar/sidebar.css.ts [hunk 3]
```diff
@@ -24,8 +24,9 @@ export const sidebar = recipe({
       bottom: 0,
       overflowY: 'scroll',
       position: 'relative',
-      top: `${config.appHeader.height}px`,
-      height: `calc(100% - ${config.appHeader.height}px)`
+      top: '-20px'
```
Comment (lukeac123) [inline packages/theme/src/sidebar/sidebar.css.ts:27]: temporary fix, work needs to be done on layouts for positioning of sidebar

File: packages/site-components/src/VerticalNavigation.tsx (hunk 1 above)
Comment (mark-tate) [reply packages/site-components/src/VerticalNavigation.tsx:21]: i think Fusion asked for it to be the same size as the content.. i removed  the action1 style we added previously.. if we need it back to 13px, have a look at the lines I removed and we can re-add/refactor as needed

**Comment by mark-tate** [comment]
looks like this PR has my old  commits on it..
Probably need to re-fetch to sync your git index.

Also worth looking at the other PR I pushed as it fixes up a few issues and give you some examples of how to use the theme to configure salt variables.
Hope it helps.

(Feel free to change)

**Pull request closed without merging**